src/Server.cpp

```diff
@@ -89,11 +89,15 @@ void Server::startPollEventLoop()
 void Server::readFromClient(Connection &conn, size_t &i, Parser &parser, HTTPRequest &request, HTTPResponse &response)
 {
 	(void)i;
-	std::cout << "\033[1;36m" << "Entering readFromClient" << "\033[0m" << std::endl;
+	std::cout << "\033[1;36m"
+			  << "Entering readFromClient"
+			  << "\033[0m" << std::endl;
 	// TODO: change to _areHeadersCopmplete
```

Comment:
Can we change these with the proper macros and check how to implement this in the Debug::log thing. Maybe linear issue about this?

Reply:
what do you mean by "implement this" you mean Debug printing from which function it was called ?
If so I just checked it and there is no good way for us